code/game/objects/items/kitchen.dm

```diff
@@ -225,4 +239,17 @@
 /obj/item/kitchen/rollingpin/suicide_act(mob/living/carbon/user)
 	user.visible_message("<span class='suicide'>[user] begins flattening [user.p_their()] head with \the [src]! It looks like [user.p_theyre()] trying to commit suicide!</span>")
 	return BRUTELOSS
-/* Trays  moved to /obj/item/storage/bag */
+
+/obj/item/kitchen/rollingpin/cyborg
+	name = "rolling pin module"
+	force = 10
+
+/obj/item/kitchen/rollingpin/cyborg/attack(mob/living/target, mob/living/carbon/human/user)
+	if (!iscyborg(user))
+		..()
+	var/mob/living/silicon/robot/R = user
+	if(!R.emagged && target.stat != DEAD)
+		playsound(src, 'sound/machines/buzz-sigh.ogg', 50, 1, -1)
```

Comment:
Dont force the Borg to be engaged for somehti g simple like hitting them with a rolling stick, all other over modules can attack